Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -123,6 +132,27 @@ extension SettingsDictionary {
     public func swiftOptimizeObjectLifetimes(_ enabled: Bool) -> SettingsDictionary {
         merging(["SWIFT_OPTIMIZE_OBJECT_LIFETIME": SettingValue(enabled)])
     }
+    
+    // MARK: - Swift Compiler - General
+
+    /// Sets `"SWIFT_OBJC_BRIDGING_HEADER"` to `path`
+    public func swiftObjcBridingHeaderPath(_ path: String) -> SettingsDictionary {
+        merging(["SWIFT_OBJC_BRIDGING_HEADER": SettingValue(path)])
+    }
+    
+    // MARK: - Apple Clang - Custom Compiler Flags
+    
+    /// Sets `"OTHER_CFLAGS"` to `flags`
+    public func otherCFlags(_ flags: String...) -> SettingsDictionary {
+        merging(["OTHER_CFLAGS": SettingValue(flags.joined(separator: " "))])
+    }
+    
+    // MARK: - Linking
+
+    /// Sets `"OTHER_LDFLAGS"` to `flags`
+    public func otherLinkerFlags(_ flags: String...) -> SettingsDictionary {
+        merging(["OTHER_LDFLAGS": SettingValue(flags.joined(separator: " "))])
+    }
```

Comment:
for those flags I believe Xcode prefers them to be declared as arrays rather inlined as a single string

something along these lines:

```swift
merging(["OTHER_LDFLAGS": .array(flags)])
```

Reply:
I think I figured it out, can you look @kwridan ?